FFCDemoPaymentService/Scheduling/SchedulerService.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using FFCDemoPaymentService.Models;
+using FFCDemoPaymentService.Data;
+
+namespace FFCDemoPaymentService.Scheduling
+{
+    public class ScheduleService : IScheduleService
+    {
+        private readonly ApplicationDbContext db;
+        public ScheduleService(ApplicationDbContext db)
+        {
+            this.db = db;
+        }
+
+        public void CreateSchedule(string claimId)
+        {
+            DateTime startDate = DateTime.Now.Date;
```

Comment:
NIT: personally I prefer for methods to be immutable.  Ie if the same parameters are passed in the method performs exactly the same action.  In this example the behaviour of the method changes depending on the timestamp.  This can be avoided by adding the start date as a parameter.

Happy to leave as is though.

Reply:
updated